lib/wordpress.js

```diff
@@ -257,6 +257,88 @@ extend( Client.prototype, {
 	}
 });
 
+extend( Client.prototype, {
+	getUsersBlogs: function( fn ) {
+		this.credentialsCall( "wp.getUsersBlogs", fn);
+	},
+
+	getUser: function( id, fields, fn ) {
+		if ( typeof fields === "function" ) {
+			fn = fields;
+			fields = null;
+		}
+
+		if ( fields ) {
+			fields = fieldMap.array( fields, "user" );
+		}
+
+		this.authenticatedCall( "wp.getUser", id, fields, function( error, user ) {
+			if ( error ) {
+				return fn( error );
+			}
+
+			fn( null, fieldMap.from( user, "user" ) );
+		});
+	},
+
+	getUsers: function ( filter, fields, fn ) {
+		if ( typeof filter === "function" ) {
+			fn = filter;
+			fields = null;
+			filter = {};
+		}
+
+		if ( typeof fields === "function" ) {
+			fn = fields;
+			fields = null;
+		}
+
+		if ( filter.orderby ) {
+			filter.orderby = fieldMap.array( [ filter.orderby ], "user" )[ 0 ];
+		}
+
+		if ( fields ) {
+			fields = fieldMap.array( fields, "user" );
+		}
```

Comment:
So much repetition (my fault, not yours). I should try to abstract out the common parts of this code. If you have any ideas for that, let me know. But don't worry about that for this PR; you've got the write approach. I'm really just writing this as a note for myself :-)